workflow/common/convert.go

```diff
@@ -11,16 +13,20 @@ func ConvertCronWorkflowToWorkflow(cronWf *wfv1.CronWorkflow) *wfv1.Workflow {
 	meta := metav1.ObjectMeta{
 		GenerateName: cronWf.Name + "-",
 		Labels:       make(map[string]string),
-		Annotations:  make(map[string]string),
+		Annotations: map[string]string{
+			AnnotationKeyCronWfScheduledTime: time.Now().Format(time.RFC3339),
```

Comment:
`time.Now()`?? Does this get overridden by `scheduledTime` as computed by the cron controller?

Reply:
yes. The default value will get override if the user passes the  --scheduledTime.